comictaggerlib/volumeselectionwindow.py

```diff
@@ -334,6 +345,40 @@ def searchComplete(self):
             return
 
         self.cv_search_results = self.search_thread.cv_search_results
+        # filter the blacklisted publishers if setting set
+        if self.use_blackList:
+            try:
+                publisher_blacklist = set([s.strip().lower() for s in self.settings.id_publisher_blacklist.split(',')])
```

Comment:
So I just discovered that there are [set comprehensions](https://dmerej.info/blog/post/hello-flake8/) you just use curly braces instead of straight braces.
```suggestion
                publisher_blacklist = {s.strip().lower() for s in self.settings.id_publisher_blacklist.split(',')}
```

Reply:
This now fails if the start_year has a non numerical value -
Comic VInes has a 'Captain Marvel' series with a start year of '1976?'

Reply:
What causes the failure this is creating a set for the publisher filter, where does the year come into play?

Reply:
Think I reported this wrong, I must have tested bad code
(need to stop dong things late a night)